components/coachmark/stories/coachmark.stories.js

```diff
@@ -63,18 +104,29 @@ export default {
 		docs: {
 			story: {
 				height: "300px",
-			}
-		}
+			},
+		},
 	},
 };
 
 export const Default = CoachMarkGroup.bind({});
 Default.title = "Standard";
 Default.tags = ["!autodocs"];
-Default.args = {};
+Default.args = {
+	title: "Coach mark title",
+	image: "example-card-landscape.png",
+	hasKeyboardShortcut: false,
```

Comment:
Could we put this up with the default args instead? Like line 91ish? that matches how other components are, at least in my head.